styles/index.css

```diff
@@ -9,3 +9,7 @@
 @tailwind components;
 /* purgecss end ignore */
 @tailwind utilities;
+
+html {
+  overflow-y: scroll;
```

Comment:
a to po co?

Reply:
bo jak się skacze pomiędzy zakładkami taski/rozwiązania to na jednym jest kontent i pojawia się scroll, który przesuwa treść strony o wielkość tego scrolla. Ten `overflow` temu przeciwdziała.